README.md

```diff
@@ -39,13 +43,23 @@ The modules in the monorepo are organized into two categories:
 - Install dependencies: `pnpm i`
 
 - Start server:
+  
   `cd apps/api`
+  
+  `cp .env.schema .env`
+
   `pnpm run dev`
+  
   The server is now listening for requests at `http://localhost:3000`
 
 - Start UI:
+
   `cd apps/ui`
+
+  `cp .env.schema .env`
+
   `pnpm run dev`
+
   Visit `http://localhost:5173/` in the browser to get started
```

Comment:
is that port chosen at random? seems atypical enough to be purposefully chosen without it following a known standard

Reply:
This is a Vite default: https://vite.dev/config/server-options#server-port
So this may end up being configured differently (and readme updated accordingly) or we could add a link to the docs